### PR TITLE
search-bar

### PR DESCRIPTION
**Pull request by Half-a-man**
- [DEMO LINK](https://Half-a-man.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Half-a-man.github.io/layout_search-bar-airbnb/report/html_report/)

Icon implemented using background-image CSS property - [x]
 Inputs are written inside of 'form' tag with correctly passed attributes - [x]
 All Typical Mistakes from BEM lesson theory are checked. - [x]
 Code follows all the [Code Style Rules ❗️] - [x]

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,36 @@
...
+    data-qa="big"
+    >
+      <label
+      class="search-form__label search-form__label_big">
+        <input
+        type="text"
+        class="search-form__input search-form__input-big"
+        name="input_big"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress">
```
Comment (volodymyr-soltys97) [inline src/index.html:25]: Need to fix code style everywhere
```suggestion
        <input
          type="text"
          class="search-form__input search-form__input-big"
          name="input_big"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
```

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="search-form search-form_big"
+    data-qa="big"
+    >
+      <label
+      class="search-form__label search-form__label_big">
```
Comment (volodymyr-soltys97) [inline src/index.html:19]:
```suggestion
      <label class="search-form__label search-form__label_big">
```

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+body {
+  font-family: "Avenir", serif;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
```
Comment (volodymyr-soltys97) [inline src/style.css:14]: Default value, so you can just remove it everywhere
```suggestion
```

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+  src: url(fonts/Avenir-Book.ttf) format("truetype");
```
Comment (volodymyr-soltys97) [inline src/style.css:15]:
```suggestion
  src: url(fonts/Avenir-Book.ttf);
```

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+@font-face {
+  font-family: "AvenirS";
+  font-weight: 300;
+  font-style: normal;
+  src: url(fonts/Avenir-Heavy.ttf) format("truetype");
```
Comment (volodymyr-soltys97) [inline src/style.css:22]:
```suggestion
  src: url(fonts/Avenir-Heavy.ttf);
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
Hey! It seems like you forgot to push your changes after the previous review (there is no new commit on github). Please, push changes and re-request review

File: src/index.html [hunk 6]
```diff
@@ -11,6 +11,34 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    class="search-form search-form_big"
+    data-qa="big"
+    >
```
Comment (volodymyr-soltys97) [inline src/index.html:17]:
```suggestion
    <form
      class="search-form search-form_big"
      data-qa="big"
    >
```

File: src/index.html [hunk 7]
```diff
@@ -11,6 +11,34 @@
...
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+        >
+      </label>
+    </form>
+
+    <form
+    class="search-form search-form_small"
+    data-qa="small"
+    >
```
Comment (volodymyr-soltys97) [inline src/index.html:32]:
```suggestion
    <form
      class="search-form search-form_small"
      data-qa="small"
    >
```

**Review by volodymyr-soltys97 — APPROVED** [review]
Good job 👍

**Pull request closed without merging**